src/Model/Todo/TodoReminder.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Prooph\ProophessorDo\Model\Todo;
+
+/**
+ * Class TodoReminder
+ *
+ * @package Prooph\ProophessorDo\Model\Todo
+ * @author Roman Sachse <r.sachse@ipark-media.de>
+ */
+final class TodoReminder
+{
+    /**
+     * @var \DateTimeImmutable
+     */
+    private $reminder;
+
+    /**
+     * @var \DateTimeImmutable
+     */
+    private $createdOn;
+
+    /**
+     * @param string $reminder
+     * @return TodoReminder
+     */
+    public static function fromString($reminder)
+    {
+        return new self($reminder);
+    }
+
+    /**
+     * @param string $reminder
+     */
+    private function __construct($reminder)
+    {
+        $this->reminder = new \DateTimeImmutable($reminder);
+        $this->createdOn = new \DateTimeImmutable;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isInThePast()
+    {
+        return $this->reminder < $this->createdOn;
+    }
+
+    /**
+     * @return string
+     */
+    public function toString()
+    {
+        return $this->reminder->format(\DateTime::ATOM);
```

Comment:
ATOM is correct. See note here: http://php.net/manual/en/class.datetime.php
ATOM is the real ISO8601 (php logic ;)), which can be handled by js/moment.js